code/common/finspace.q

```diff
@@ -39,8 +39,8 @@ notifyhdb:{[cluster;changeset]
       // Ensuring that the changeset has successfully created before doing the HDB reload
       current:.finspace.checkstatus[(`.aws.get_changeset;.finspace.database;changeset[`id]);("COMPLETED";"FAILED");00:01;0wu];
       .lg.o[`notifyhdb;("notifying ",string[cluster]," to repoint to changeset ",changeset[`id])];
-      awsdb:.aws.db[.finspace.database;changeset[`id];.aws.cache["CACHE_1000";"/"];""];
-      .aws.update_kx_cluster_databases[string[cluster];.aws.sdbs[awsdb];.aws.sdep["NO_RESTART"]]
+      awsdb:.aws.db[.finspace.database;changeset[`id];();"dndataview"];
+      .aws.update_kx_cluster_databases[string[cluster];.aws.sdbs[awsdb];.aws.sdep["ROLLING"]]
```

Comment:
have we concluded we will only be supporting ROLLING deployment strategies?

Reply:
Yeah, if nothing's been concluded yet, best to keep it as is (within the master branch) or add an if-else statement. It's a bit of a pain, but you can update the function to explicitly use ROLLING in your local branches

Reply:
No worries that makes sense, I'll put it back to ROLLING for the meantime.

Reply:
Sorry, i mean NO_RESTART